nextflow.config

```diff
@@ -0,0 +1,45 @@
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+nextflowVersion = '!>=22.11'
+
+plugins {
+    id 'nf-validation@1.1.3'
+}
+
+params {
+    query = ""
+    species = ""
+    seq_type = ""
+    refseq = false
```

Comment:
Why it is a boolean now? Before changes, ref_seq was added to the query as: " AND ${params.ref_seq}[filter]"

Reply:
It is still added in the same fashion, the only thing that has changed is that before it had to be passed with a 0-1 value in the command line (`--ref_seq 1`), and now it behaves like a flag (`--refseq`).

Reply:
I believe that was a mistake on their side. The sequence type (`--seq_type`) is the one that should be used for `mitochondrion`, and `--refseq` is to filter only the reference sequences or fetch all available.

Reply:
Oh, I see... I will change the GUI to fit with that, so that now it is chosen or not, without any handwritten input.